components/rhs_thread/index.ts

```diff
@@ -25,24 +20,17 @@ function makeMapStateToProps() {
 
     return function mapStateToProps(state: GlobalState) {
         const selected = getSelectedPost(state);
-        const socketStatus = getSocketStatus(state);
 
         const channel = getSelectedChannel(state);
         let posts: Post[] = [];
         if (selected) {
             posts = getPostsForThread(state, {rootId: selected.id});
         }
 
-        const previewCollapsed = get(state, Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.COLLAPSE_DISPLAY, Preferences.COLLAPSE_DISPLAY_DEFAULT);
-
         return {
             selected,
             channel,
             posts,
-            socketConnectionStatus: socketStatus.connected,
```

Comment:
None of these props are actually used by `RhsThread`, so they were safe to just remove.